helm-chart/ray-operator-helm/values.yaml

```diff
@@ -0,0 +1,47 @@
+# Default values for ray-operator-helm.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: docker.io/library/controller
```

Comment:
can you check image from current master for the default value?

Reply:
I run "make docker-build" command, and it shows me "naming to docker.io/library/controller:latest". Is this the default value?

Reply:
Got you. We normally use `IMG=xxx make docker-build` to override the IMG. I think we have not updated Makefile and that's why you get `docker.io/library/controller:latest`